src/restaurants/restaurants.js

```diff
@@ -6,65 +8,71 @@ export function initializeRestaurants() {
-      restaurantLongitude.innerHTML = "Longitude: " + result.longitude;
-
-      div.appendChild(restaurantId);
-      div.appendChild(restaurantName);
-      div.appendChild(restaurantAddress1);
-      div.appendChild(restaurantAddress2);
-      div.appendChild(restaurantLatitude);
-      div.appendChild(restaurantLongitude);
-      document.getElementById("main").appendChild(div);
-
-      let div2 = document.createElement("div");
-      const button = document.createElement("button");
-      button.type = "button";
-      button.className = "button button";
-      button.innerHTML = "Menu";
-
-      let hasButtonBeenClicked = false;
-
-      //click button only one time
-      button.addEventListener("click", function () {
-        if (!hasButtonBeenClicked) {
-          getRestaurantMenu();
-          hasButtonBeenClicked = true;
-        }
-      });
-
-      div2.appendChild(button);
-      document.getElementById("main").appendChild(div2);
+  fetchData(
+    `https://private-anon-7231255228-pizzaapp.apiary-mock.com/restaurants/${id}`
+  ).then((result) => {
+    // let div = document.createElement("div");
+
+    // const restaurantId = document.createElement("tr");
+    // restaurantId.innerHTML = "Id: " + result.id;
+    // const restaurantName = document.createElement("tr");
+    // restaurantName.innerHTML = "Name: " + result.name;
+    // const restaurantAddress1 = document.createElement("tr");
+    // restaurantAddress1.innerHTML = "Address1: " + result.address1;
+    // const restaurantAddress2 = document.createElement("tr");
+    // restaurantAddress2.innerHTML = "Address2: " + result.address2;
+    // const restaurantLatitude = document.createElement("tr");
+    // restaurantLatitude.innerHTML = "Latitude: " + result.latitude;
+    // const restaurantLongitude = document.createElement("tr");
+    // restaurantLongitude.innerHTML = "Longitude: " + result.longitude;
+
+    // div.appendChild(restaurantId);
+    // div.appendChild(restaurantName);
+    // div.appendChild(restaurantAddress1);
+    // div.appendChild(restaurantAddress2);
+    // div.appendChild(restaurantLatitude);
+    // div.appendChild(restaurantLongitude);
+    // document.getElementById("main").appendChild(div);
+
+    let div = document.createElement("div");
+
+    div.innerHTML = `
+    <h3>${result.name}</h3>
+    <p><strong>Address I:</strong> ${result.address1}</p>
+    <p><strong>Address II:</strong>${result.address2}</p>
+    <p><strong>Latitude:</strong> ${result.latitude}</p>
+    <p><strong>Longitude:</strong> ${result.longitude}</p>`;
+
+    document.getElementById("main").appendChild(div);
+
+    let div2 = document.createElement("div");
+    const button = document.createElement("button");
+    button.type = "button";
+    button.className = "button button";
+    button.innerHTML = "Menu";
+
+    let hasButtonBeenClicked = false;
+
+    //click button only one time
+    button.addEventListener("click", function () {
+      if (!hasButtonBeenClicked) {
```

Comment:
If this is needed then we should use this variable (flag) to disable the button once it's clicked, this way this check is not needed.